apt-requirements.txt

```diff
@@ -7,6 +7,7 @@ autoconf
 bison
 build-essential
 clang-format
+cmake
```

Comment:
Is this left over from the combination of you and @alphan's work?

Reply:
This will now conflict. I landed a change to update `apt-requirements.txt` and the online documents this after googletest was vendored (building googletest uses CMake).

Reply:
This is a fixup for this being missing from @alphan's PR isn't it?  Can you split this out to a new PR?

Reply:
Yeah, the new PR was #1446 and it has already been merged.